README.md

```diff
@@ -40,6 +40,8 @@ or
 ```
 cmake --build . --config Release --target VocalTractLabBackend
 ```
+The final dll/so-files and executables are copied to ``../lib/`` so you don't
```

Comment:
Revise to:

The final binary files are placed into the subdirectory (of the repository root) ``lib``. Do not look for them in the ``out`` directory, which is only used as a temporary folder for the build process and can be safely deleted once the binaries are built.